conans/test/unittests/tools/files/test_downloads.py

```diff
@@ -77,6 +77,27 @@ def test_download_forbidden(self, _manual):
             download(conanfile, file_server.fake_url + "/forbidden", dest)
         assert "403 Forbidden" in str(exc.value)
 
+    def test_download_unauthorized_no_credentials(self, _manual):
```

Comment:
Excellent. Adding a new test instead of modifying existing ones is generally better, nice one.

Reply:
Thank you!